src/Region/Ds9ImportExport.cc

```diff
@@ -503,34 +497,24 @@ RegionState Ds9ImportExport::ImportEllipseRegion(std::vector<std::string>& param
         // Control points in pixel coordinates
         std::vector<CARTA::Point> control_points;
         if (_pixel_coord) {
-            CARTA::Point point;
-            point.set_x(param_quantities[0].getValue());
-            point.set_y(param_quantities[1].getValue());
-            control_points.push_back(point);
-            point.set_x(param_quantities[2].getValue());
-            point.set_y(param_quantities[3].getValue());
-            control_points.push_back(point);
+            control_points.push_back(Message::Point(param_quantities[0].getValue(), param_quantities[1].getValue()));
+            control_points.push_back(Message::Point(param_quantities[2].getValue(), param_quantities[3].getValue()));
         } else {
             // cx, cy
             std::vector<casacore::Quantity> center_coords;
             center_coords.push_back(param_quantities[0]);
             center_coords.push_back(param_quantities[1]);
             casacore::Vector<casacore::Double> pixel_coords;
             if (ConvertPointToPixels(_file_ref_frame, center_coords, pixel_coords)) {
-                CARTA::Point point;
-                point.set_x(pixel_coords(0));
-                point.set_y(pixel_coords(1));
-                control_points.push_back(point);
+                control_points.push_back(Message::Point(pixel_coords(0), pixel_coords(1)));
             } else {
                 _import_errors.append("Failed to apply ellipse to image.\n");
                 return region_state;
             }
 
             // bmaj, bmin
-            CARTA::Point point;
-            point.set_x(WorldToPixelLength(param_quantities[2], 0));
-            point.set_y(WorldToPixelLength(param_quantities[3], 1));
-            control_points.push_back(point);
+            control_points.push_back(
+                Message::Point(WorldToPixelLength(param_quantities[2], 0), WorldToPixelLength(param_quantities[3], 1)));
```

Comment:
It's probably not worth doing anything about this case; we can keep it like this.